tests/chart_tests/test_grafana.py

```diff
@@ -105,3 +105,16 @@ def test_deployment_with_securitycontext_overrides(self, kube_version):
             "runAsNonRoot": True,
             "runAsUser": 467,
         }
+
+    def test_grafana_init_containers_disabled_with_custom_secret_name(self, kube_version):
+        """Test that the grafana deployment init containers disabled."""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={"grafana": {"backendSecretName": "grafanabackend"}},
```

Comment:
We don't appear to have any tests that ensure `backendSecretName` is working the way we expect it to.